.github/workflows/cxx-build.yml

```diff
@@ -11,23 +11,31 @@ on:
       - opened
       - synchronize
 
+defaults:
+  run:
+    shell: bash
+
 env:
   HUSKY: 0
 jobs:
   thunder:
     name: Build Thunder Libraries
     runs-on: ubuntu-latest
     container:
-      image: kevinshahfws/node-c:3.0
+      image: node:latest
```

Comment:
Can we manually specify whatever the latest version is here? It wouldn't be the first time that "latest" changed to something that broke a pipeline.

Reply:
fixed